modules/cli/cli.go

```diff
@@ -44,6 +44,9 @@ func New(m metadata.Info) *CLI {
 	// controllers for specialized APIs and features
 	flagSet.BoolVar(&cfg.AIGatewayControllerEnabled, "enable-controller-aigateway", false, "Enable the AIGateway controller. (Experimental).")
 
+	// controllers for Konnect APIs
+	flagSet.BoolVar(&cfg.KonnectControllersEnabled, "enable-controller-konnect", false, "Enable the KonnectAPIAuthConfiguration controller.")
```

Comment:
I think we should enable/disable all the konnect-related controllers based on this flag, not only the `konnectAPIAuthConfiguration` one. What do you think?

Reply:
Argh, this is what the flag does (in https://github.com/Kong/gateway-operator/pull/456/files#diff-21545a58e07f7be7b539e19a03651f2f7fed85ad4cbd1fc72d15cdb765888706R277-R288) but I forgot to change the description. 

269dc5a3379c3996e5844eca78873f1c82514707 does just that.